assets/resources/badusb/Install_qFlipper_windows.txt

```diff
@@ -0,0 +1,36 @@
+REM Written by @dexv
+DELAY 2000
+GUI r
+DELAY 500
+STRING powershell
+ENTER
+DELAY 1000
+STRING $url = "https://update.flipperzero.one/qFlipper/release/windows-amd64/portable"
+ENTER
+STRING $output = "$env:USERPROFILE\Documents\qFlipper.zip"
+ENTER
+STRING $destination = "$env:USERPROFILE\Documents\qFlipper"
+ENTER
+STRING $shortcutPath = "$env:USERPROFILE\Desktop\qFlipper.lnk"
+ENTER
+STRING $scriptPath = "$env:USERPROFILE\Documents\qFlipperInstall.ps1"
+ENTER
+STRING (New-Object System.Net.WebClient).DownloadFile($url, $output)
+ENTER
+STRING Expand-Archive -Path $output -DestinationPath $destination -Force
+ENTER
+STRING Set-Location -Path $destination
+ENTER
+STRING Start-Process -FilePath ".\qFlipper.exe"
```

Comment:
Note that since the automatic installer is not used, the DFU driver needs to be installed manually. (It would require admin rights.) Otherwise, the repair functionality will not be available, should the script be run on a system with no previous qFlipper installations.
The driver is located in the `STM2 Driver` directory in the archive, and the installation procedure is specified in `install.bat` in the same folder.

Reply:
Ah I see let me see what i can do